### PR TITLE
Remove stats_solr as part of our migration to ol-home0

### PR DESCRIPTION
**Pull request by cclauss**
Related to #7817

The files `openlibrary/views/loanstats.py` and the related sections of `openlibrary/templates/home/index.html` were not modified because they pertain more to `trending_books` rather than `loanstats`.

### CODE REVIEW

File: openlibrary/core/loanstats.py [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
-import logging
-import requests
-from urllib.parse import urlencode
-from infogami import config
-
-
-logger = logging.getLogger(__name__)
-
-
-class LoanStats:
```
Comment (cclauss) [inline openlibrary/core/loanstats.py:14]: Remove the class and then do local tests....,

File: openlibrary/plugins/openlibrary/api.py [hunk 2]
```diff
@@ -62,37 +61,6 @@ def get_book_availability(self, id_type, ids):
         )
 
 
-class trending_books_api(delegate.page):
```
Comment (cclauss) [inline openlibrary/plugins/openlibrary/api.py:65]: Let's keep this!

File: openlibrary/solr/process_stats.py [hunk 3]
```diff
@@ -1,271 +0,0 @@
-"""Script to process various stats collected by the system and load them into solr.
```
Comment (cclauss) [inline openlibrary/solr/process_stats.py:1]: Let's make sure that the deleted code is ONLY extracting things from the db2 solr.  Any code that writes data to db should be preserved.

**Review by cdrini — APPROVED** [review]
Lgtm! All looks like dead code

**Pull request merged**